CreatePackages.md

```diff
@@ -202,12 +202,12 @@ To test the package you just built, open a command line shell and navigate to th
 
 PowerShell:
 ```powershell
- choco install packageName -fdv -s '$pwd'
+ choco install packageName -fdv -s "$pwd"
```

Comment:
Will take your word for it - I have no idea when to quote things or not in posh.